Add tuple return type

evanjhoward11 opened the pull request:
Add tuple as a return type to get_wdl_stats.
Add one test to ensure return type is a tuple when argument get_as_tuple=True.
Closes issue #46.

johndoknjas commented:
Thanks for your PR @evanjhoward11! I've added a few suggestions but nothing major.

johndoknjas commented on stockfish/models.py:
```diff
@@ -706,11 +706,18 @@ def is_move_correct(self, move_value: str) -> bool:
-    def get_wdl_stats(self) -> Optional[List[int]]:
+    def get_wdl_stats(
+        self, get_as_tuple: bool = False
+    ) -> Union[list[int] | tuple[int, ...] | None]:
```
For this we could do `tuple[int, int, int]`, since it's guaranteed the tuple will have 3 elements.

johndoknjas commented on tests/stockfish/test_models.py:
```diff
@@ -914,6 +914,9 @@ def test_get_wdl_stats(self, stockfish: Stockfish):
 
+            wdl_stats_4 = stockfish.get_wdl_stats(get_as_tuple=True)
+            assert isinstance(wdl_stats_4, tuple) and len(wdl_stats_4) == 3
+
```
Here we could also test that `tuple(wdl_stats_3)` equals `wdl_stats_4`, and also `list(wdl_stats_4)` equals `wdl_stats_3`. From the changes you've made this should clearly be the case, but in the future the test will serve as a regression test in case a bad change is made to the `get_wdl_stats` function.

kieferro reviewed (approved):
Thanks for the PR. Except for what @johndoknjas has already commented, everything looks good.

evanjhoward11 commented:
Sorry for the late reply. I added the necessary changes that @johndoknjas mentioned. Not sure why the Linux-related tests are failing. Perhaps it has something to do with the files I added to my local folder for testing purposes (main.py, stockfish exe, and .gitignore)?
